tests/pos/i4226.scala

```diff
@@ -0,0 +1,15 @@
+sealed abstract class Maybe[A]
```

Comment:
Is it really useful to have the same test in pos and patmat? Aren't the patmat tests also fully compiled?

Reply:
OK, let's remove it. I was thinking more in terms of features.